### PR TITLE
LPD-28653 Implemented the posiblity to download the license agreement

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

5 files changed, 92 insertions(+), 48 deletions(-)

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts [hunk 1]
```diff
@@ -644,6 +644,17 @@ type UserAccount = {
...
+		telephones: [
+			{
+				extension?: string;
+				id?: number;
+				phoneNumber?: string;
+				phoneType?: string;
+				primary?: boolean;
+			},
+		];
+	};
```
Comment (joanagrego) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts:657]:
```suggestion
type UserAccountTelephone = {
				extension?: string;
				id?: number;
				phoneNumber?: string;
				phoneType?: string;
				primary?: boolean;
			};

type UserAccountContactInfo = {
				telephones: UserAccountTelephone[]
			};
			
	type UserAccount = {
	accountBriefs: AccountBrief[];
	alternateName: string;
	currentPassword: string;
	emailAddress: string;
	externalReferenceCode: string;
	familyName: string;
	givenName: string;
	id: number;
	image: string;
	isCustomerAccount: boolean;
	isPublisherAccount: boolean;
	logoURL: string;
	name: string;
	newsSubscription: boolean;
	password: string;
	roleBriefs: {id: number; name: string}[];
	type: string;
	userAccountContactInformation?: UserAccountContactInfo;
};
```

**Review by joanagrego — APPROVED** [review]
Good job!

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/utils/getSite.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import {isCloudEnvironment} from './util';
+
+const getSiteName = () => {
+	const isCloundEnvironment = isCloudEnvironment();
+	const marketplaceUrl = (
+		isCloundEnvironment
+			? Liferay.ThemeDisplay.getURLHome()
+			: Liferay.ThemeDisplay.getLayoutRelativeURL()
+	).split('/');
+	const index = isCloundEnvironment ? -1 : 2;
```
Comment (joanagrego) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/utils/getSite.ts:16]: Can we rename these magic numbers?

**Review by joanagrego — APPROVED** [review]

**Comment by welitonpe** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]

**Comment by welitonpe** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**